lib/manager/components/HomeProjectDropdown.js

```diff
@@ -17,20 +18,9 @@ type Props = {
   visibleProjects: Array<Project>
 }
 
-type State = {
-  selectedOption: any
-}
-
-export default class HomeProjectDropdown extends Component<Props, State> {
-  state = {
-    selectedOption: ''
-  }
-
-  handleChange = (selectedOption: any, activeProject: any) => {
-    this.setState({selectedOption})
-    activeProject === null
-      ? browserHistory.push(`/home/${selectedOption.value}`)
-      : browserHistory.push(`/home/`)
+export default class HomeProjectDropdown extends Component<Props> {
+  handleChange = (option: ReactSelectOption) => {
+    browserHistory.push(`/home/${option ? option.value : ''}`)
```

Comment:
Moving the ternary down the chain into the string construction allows us to only invoke `browserHistory.push` once. Ultimately, the same code, but I find this a bit more clear. Another nice way to frame this might be:

```javascript
  handleChange = (option: ReactSelectOption) => {
    const path = option ? '/home/' : `/home/${option.value}`
    browserHistory.push(path)
  }
```